dali/operators/image/convolution/gaussian_blur.cc

```diff
@@ -138,11 +138,11 @@ class GaussianBlurOpCpu : public OpImplBase<CPUBackend> {
         thread_pool.AddWork(
             [this, &input, &output, sample_idx, elem_idx, stride](int thread_id) {
               auto gaussian_windows = windows_[sample_idx].GetWindows();
-              auto elem_shape = input[sample_idx].shape().template last<ndim>();
+              auto elem_shape = input.tensor_shape(sample_idx).template last<ndim>();
```

Comment:
Pff... this will conflict with per-sample argument PR.

Reply:
As will the changes in TensorVector - we will resolve.